### PR TITLE
Vscode themes branch

### PR DESCRIPTION
**Pull request by irinaponomarevahaulmont**
Please review the original draft

### CODE REVIEW

File: content/modules/vscode-extension/pages/vscode-themes.adoc [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+[TIP]
+Unlike IntelliJ IDEA, VS Code doesn't save the files automatically. You have to remember to save your modified files manually to avoid losing the changes.
+
+We have changed first three color masks, and now the theme looks as follows:
+
+image::themes/jelly-theme-modified.png[align=center, width=500]
+
+Parameter CSS styles, font parameters, margins, etc., can also be changed in a similar way.
+
+[TIP]
```
Comment (alexbudarov) [inline content/modules/vscode-extension/pages/vscode-themes.adoc:146]: The same tip twice, so near to each other?

**Review by alexbudarov — CHANGES_REQUESTED** [review]

**Comment by irinaponomarevahaulmont** [comment]
Please review the changes

File: content/modules/vscode-extension/pages/vscode-themes.adoc [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+. The extension offers a UI for displaying the list of themes available within the project.
+
+image::themes/amplicode-custom-theme.png[align=center, width=300]
+
+[start=3]
+
+. There is a theme preview feature (image:themes/preview.png[]).
+
+. Users can navigation to the theme definition file to edit it (image:themes/edit.png[]).
```
Comment (alexbudarov) [inline content/modules/vscode-extension/pages/vscode-themes.adoc:17]: can navigate ?

File: content/modules/vscode-extension/pages/vscode-themes.adoc [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+* *_Edit theme_* (image:themes/edit.png[]) -- this action opens the Typescript file where the selected theme is defined (the theme's main configuration file), and this action is available for custom themes only; it allows us to modify the main configuration file of the selected theme in an editor in order to customize the theme
+
+* *_Delete theme_* (image:themes/delete.png[]) is the action, which deletes the selected theme from the project along with all the files that belong to it; available for custom themes only
+
+* The *_Create new theme_* button allows us to create a new custom theme.
+
+image::themes/create-own-theme.png[align=center, width=300]
+
+Once this action is completed, a set of files gets created, which defines the theme. The created theme becomes available for using in the project *and customization*.
```
Comment (alexbudarov) [inline content/modules/vscode-extension/pages/vscode-themes.adoc:86]: I didn't mean to write it in bold. I was just emphasising it in the comment.

**Review by alexbudarov — CHANGES_REQUESTED** [review]

**Comment by irinaponomarevahaulmont** [comment]
Please review second round of changes.

**Pull request merged**